packages/typography/src/Error/Error.tsx

```diff
@@ -20,6 +26,7 @@ const Error = Polymorphic<BaseErrorProps>(
     ...rest
   }) => {
```

Comment:
what was this defaulting to before? and are we sure this is what all the designs call for? i feel like i remember error having its own typography rules but i could be wrong

Reply:
It was defaulting to `inherit` before, so 16px unless a parent set a font-size